File: lista1_js/29.meses.js
//29. Leia um número inteiro de meses, calcule e escreva quantos anos e quantos meses ele corresponde.

import { question } from "readline-sync"

//Entrada
const meses = Number(question('Informe a quantidade de meses: '))


//Processamento
const anos = Math.floor(meses / 12)
const mes = meses % 12

//Saída
console.log(`\n>>> O equivalente a essa quantidade de meses é de ${anos.toFixed(0)} ano(s) e ${mes.toFixed(0)} mes(es).`) 

File: lista1_py/29.meses.py
#29. Leia um número inteiro de meses, calcule e escreva quantos anos e quantos meses ele corresponde.

#Entrada
meses = int(input('Informe a quantidade de meses: '))

#Processamento
anos = meses // 12
mes = meses % 12

#Saída
print(f'\n>>> O equivalente a essa quantidade de meses é de {anos} ano(s) e {mes} mes(es).')